
function addUser() {

}

try {
  module.exports = {
    addUser: addUser
  }
} catch (error) {
  console.log('we caught an error');
}
